test/witan/app/model_execution_test.clj

```diff
@@ -16,7 +27,7 @@
    :name "My Forecast 1"
    :created #inst "2015-10-30T17:51:21.244-00:00"
    :inputs {"Development data" {:data_id development-data-id
-                                :category "Development data"
+                                :category input-category-a
```

Comment:
I think this should be `(:category input-category-a)` (and same for all data items in this file)